src/sortStudents.ts

```diff
@@ -1,16 +1,91 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function getAverageGrade(student: Student):number {
+  const gradesSum = student.grades.reduce(
+    (total :number, grade :number) => total + grade,
+  );
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  return gradesSum / student.grades.length;
+}
+
+export function sortStudents(
+  students: Student[], sortBy: SortType, order: SortOrder,
+) :Student[] {
+  const studentsCopied: Student[] = students;
+
+  switch (sortBy) {
+    case ('name'):
```

Comment:
use enum values here
```suggestion
    case SortType.Name:
```